flutter/lib/src/default_integrations.dart

```diff
@@ -44,16 +44,31 @@ class FlutterErrorIntegration extends Integration<SentryFlutterOptions> {
   void call(Hub hub, SentryFlutterOptions options) {
     _defaultOnError = FlutterError.onError;
     _integrationOnError = (FlutterErrorDetails errorDetails) async {
-      dynamic exception = errorDetails.exception;
+      final exception = errorDetails.exception;
 
       options.logger(
         SentryLevel.debug,
         'Capture from onError $exception',
       );
 
       if (errorDetails.silent != true || options.reportSilentFlutterErrors) {
+        final context = errorDetails.context?.toDescription();
+
+        final collector = errorDetails.informationCollector?.call() ?? [];
+        final information =
+            (StringBuffer()..writeAll(collector, '\n')).toString();
+        final library = errorDetails.library;
+
         // FlutterError doesn't crash the App.
-        final mechanism = Mechanism(type: 'FlutterError', handled: true);
+        final mechanism = Mechanism(
+          type: 'FlutterError',
+          handled: true,
+          data: {
+            if (context != null) 'context': context,
```

Comment:
As per [docs](https://api.flutter.dev/flutter/foundation/FlutterErrorDetails/context.html) this is a message which should make sense if written after the word `thrown`.
```suggestion
            if (context != null) 'context': 'thrown $context',
```
or maybe 
```suggestion
            if (context != null) 'thrown': context,
```

Reply:
Then what about adding ur comment to it too:

```suggestion
            // This is a message which should make sense if written after the word `thrown`: https://api.flutter.dev/flutter/foundation/FlutterErrorDetails/context.html
            if (context != null) 'context': 'thrown $context',
```